src/nnet3/nnet-test-utils.cc

```diff
@@ -804,6 +804,18 @@ static void GenerateRandomComponentConfig(std::string *component_type,
       os << "input-dim=" << input_dim << " output-dim=" << output_dim;
       break;
     }
+    case 17: {
+      *component_type = "Convolutional1dComponent";
+      int32 patch_stride = 10, patch_step = 1, patch_dim = 4;
```

Comment:
I think it would be better to test a variety of values here- this may be masking bugs.